python_modules/dagster/dagster/core/definitions/decorators/schedule_decorator.py

```diff
@@ -69,10 +69,9 @@ def schedule(
     argument, and does one of the following:
 
     1. Return a `RunRequest` object.
-    2. Yield multiple of `RunRequest` objects.
-    3. Return or yield a `SkipReason` object, providing a descriptive message of why no runs were
-       requested.
-    4. Return or yield nothing (skipping without providing a reason)
+    2. Return a list of `RunRequest` objects.
+    3. Return a `SkipReason` object, providing a descriptive message of why no runs were requested.
+    4. Return nothing (skipping without providing a reason)
```

Comment:
should we still mention that you can yield?

Reply:
I guess so... thought it was a simpler story to eventually deprecate it, but might as well document it if we still support it.